hubblestack/files/hubblestack_nova/stat_nova.py

```diff
@@ -177,3 +209,75 @@ def _get_tags(data):
                     formatted_data.pop('data')
                     ret[tag].append(formatted_data)
     return ret
+
+def _check_mode(  max_permission , given_permission, allow_more_strict):
```

Comment:
[lint] Please remove the two spaces after the opening parens, and remove single space before comma